-Force parameter on line 62 crashes the script

mioteg opened the pull request:
The -Force parameter on line 62 crashes the script with the following error:
A parameter cannot be found that matches parameter name 'Force'.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
-Force parameter on line 62 crashes the script

The -Force parameter on line 62 crashes the script with the following error:
A parameter cannot be found that matches parameter name 'Force'.
